tlslite/tlsconnection.py

```diff
@@ -1323,6 +1327,15 @@ def _clientTLS13Handshake(self, settings, session, clientHello,
                                                     self._handshake_hash,
                                                     prfName)
 
+        # TLS1.3 log Client and Server traffic secrets for SSLKEYLOGFILE
+        if self.sslkeylogfile:
```

Comment:
I think it would be cleaner to have the check if we should log stuff _inside_ the method to log it...